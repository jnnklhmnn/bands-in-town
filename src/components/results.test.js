import React from 'react';
import Results from './results.js';
import renderer from 'react-test-renderer';

test('Results should match Snapshot', () => {

    const mockinfo = {
        name: 'foo',
        thumb_url: 'http://foo.bar.de'
    }

    const mockevents = [
        {
            "id": "foo",
            "datetime": "2018-08-04T20:00:00",
            "venue": {
                "name": "The Theater at MGM National Harbor",
                "latitude": "38.791707",
                "longitude": "-77.003002",
                "city": "National Harbor",
                "region": "MD",
                "country": "United States"
            }
        },
        {
            "id": "bar",
            "datetime": "2018-08-05T20:00:00",
            "venue": {
                "name": "The Theater at MGM National Harbor",
                "latitude": "38.791707",
                "longitude": "-77.003002",
                "city": "National Harbor",
                "region": "MD",
                "country": "United States"
            }
        }
    ];

    const component = renderer.create(
        <Results info={mockinfo} events={mockevents} />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});