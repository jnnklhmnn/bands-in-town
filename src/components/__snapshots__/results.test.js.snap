// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results should match Snapshot 1`] = `
<div
  className="wrapper"
>
  <div
    className="flexwrapper"
  >
    <div
      className="left"
    >
      <div
        className="contentbox"
      >
        <div
          className="contentboxwrapper"
        >
          <div
            className="name"
          >
            <h2>
              foo
            </h2>
          </div>
          <div
            className="imagewrapper"
          >
            <img
              alt="foo"
              className="image"
              src="http://foo.bar.de"
            />
          </div>
          <a
            className="facebooklink"
            href={undefined}
            title="Visit foo on Facebook"
          >
            <span
              className="linkcopywrapper"
            >
              <span
                className="linkcopy"
              >
                Visit 
                foo
                 on
                <svg
                  className="facebookicon"
                  viewBox="0 0 1792 1792"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M1579 128q35 0 60 25t25 60v1366q0 35-25 60t-60 25h-391v-595h199l30-232h-229v-148q0-56 23.5-84t91.5-28l122-1v-207q-63-9-178-9-136 0-217.5 80t-81.5 226v171h-200v232h200v595h-735q-35 0-60-25t-25-60v-1366q0-35 25-60t60-25h1366z"
                  />
                </svg>
              </span>
            </span>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      className="contentbox"
    >
      <div
        className="youtubeheader"
      >
        <h3>
          Youtube Results
        </h3>
        <svg
          fill="#ff0000"
          height="50"
          viewBox="0 0 1792 1792"
          width="50"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M711 1128l484-250-484-253v503zm185-862q168 0 324.5 4.5t229.5 9.5l73 4q1 0 17 1.5t23 3 23.5 4.5 28.5 8 28 13 31 19.5 29 26.5q6 6 15.5 18.5t29 58.5 26.5 101q8 64 12.5 136.5t5.5 113.5v176q1 145-18 290-7 55-25 99.5t-32 61.5l-14 17q-14 15-29 26.5t-31 19-28 12.5-28.5 8-24 4.5-23 3-16.5 1.5q-251 19-627 19-207-2-359.5-6.5t-200.5-7.5l-49-4-36-4q-36-5-54.5-10t-51-21-56.5-41q-6-6-15.5-18.5t-29-58.5-26.5-101q-8-64-12.5-136.5t-5.5-113.5v-176q-1-145 18-290 7-55 25-99.5t32-61.5l14-17q14-15 29-26.5t31-19.5 28-13 28.5-8 23.5-4.5 23-3 17-1.5q251-18 627-18z"
          />
        </svg>
      </div>
      <div
        className="youtubebody"
      >
        <ul>
          <li>
            <div
              className="spinner"
            >
              <div
                className="rect1"
              />
              <div
                className="rect2"
              />
              <div
                className="rect3"
              />
              <div
                className="rect4"
              />
              <div
                className="rect5"
              />
            </div>
          </li>
        </ul>
      </div>
    </div>
  </div>
  <div>
    <div
      className="contentbox"
    >
      <div
        className="headerwrapper"
      >
        <h3
          className="name"
        >
          Go See 
          foo
           live!
        </h3>
        <img
          alt="foo"
          className="image"
          src="http://foo.bar.de"
        />
      </div>
      <ul
        className="list"
      >
        <li
          className="listitem"
        >
          <span
            className="left"
          >
            <p
              className="listitemcopy"
            >
              <span
                className="month"
              >
                aug
              </span>
               
              <span
                className="day"
              >
                 
                04
              </span>
            </p>
          </span>
          <span
            className="right"
          >
            <p
              className="listitemcopy"
            >
              The Theater at MGM National Harbor
            </p>
            <p
              className="listitemcopy"
            >
              National Harbor
            </p>
            <p
              className="listitemcopy"
            >
              United States
            </p>
          </span>
        </li>
        <li
          className="listitem"
        >
          <span
            className="left"
          >
            <p
              className="listitemcopy"
            >
              <span
                className="month"
              >
                aug
              </span>
               
              <span
                className="day"
              >
                 
                05
              </span>
            </p>
          </span>
          <span
            className="right"
          >
            <p
              className="listitemcopy"
            >
              The Theater at MGM National Harbor
            </p>
            <p
              className="listitemcopy"
            >
              National Harbor
            </p>
            <p
              className="listitemcopy"
            >
              United States
            </p>
          </span>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
